environment/README.md

```diff
@@ -6,3 +6,15 @@ See:
 - [`local-mongo`](./local-mongo)
 - [`local-tar`](./local-tar)
 - [`remote-gcp-cloudrun`](./remote-gcp-cloudrun)
+
+## Ephemeral environments
+
+Ephemeral environments are environments that are not meant to be used for a long time. They are meant to be used for development purposes only (e.g. for bugfix/feature testing or demo).
```

Comment:
Can we simplify the sentence to not repeat "they are meant to". "Ephemeral environment" is used multiple time as well. I guess it can be simplified / rephrased
<img width="885" alt="Screenshot 2023-10-17 at 14 25 39" src="https://github.com/websight-io/starter/assets/4356553/b2ecbc33-de32-4b2b-8231-2241434ab0b6">

Reply:
Docs updated